src/Layer/LabelLayer.js

```diff
@@ -34,6 +152,9 @@ class LabelLayer extends Layer {
      * contains three elements `name, protocol, extent`, these elements will be
      * available using `layer.name` or something else depending on the property
      * name.
+     * @param {boolean} [config.performance=false] - remove labels that have no chance of being visible.
+     * Indeed, even in the best case, labels will never be displayed. By example, if there's many labels.
+     * @param {boolean} [config.forceClampToTerrain=false] - use elevation layer to clamp label on terrain.
```

Comment:
Does this have an impact on performances? If yes, we should precise it in the doc. What happens if we don't set it to true? The labels may be under the terrain?

Reply:
i add comment on documentation.

> The labels may be under the terrain?

For the moment, these labels have not yet and never been managed by itowns